### PR TITLE
Update "Changing Operations [...]" section

### PR DESCRIPTION
**Pull request by yivi**
[This section](https://api-platform.com/docs/core/openapi/#changing-operations-in-the-openapi-documentation) uses a configuration format that works for version 2 of OpenApi, which clashes with the default for the distribution (v3). The documentation example simply does not work out of the box.

Users updating are also being [hit by this](https://github.com/api-platform/core/issues/4035).

Included an example with attributes, instead of annotations, since [we should always use the latest version of dependencies on documentation](https://github.com/api-platform/docs/issues/1272#issuecomment-773178233).

### CODE REVIEW

**Comment by yivi** [comment]
The new `swagger-ui-2.png` is 100kb, because it's a bit taller to show the "example value" section. It's a bit heavy. A WebP version would be only 20kb, but since all other images are PNG I didn't want to add an image in a different version.

File: core/openapi.md [hunk 1]
```diff
@@ -311,58 +311,93 @@ You also have full control over both built-in and custom operations documentatio
...
+                example:
+                  name: Mr. Rabbit
+                  description: Pink rabbit
+
 ```
 
 ```xml
 <?xml version="1.0" encoding="UTF-8" ?>
-<resources xmlns="https://api-platform.com/schema/metadata"
+<resources
```
Comment (alanpoulain) [inline core/openapi.md:384]: Why removing the `xmlns`?

File: core/openapi.md [hunk 2]
```diff
@@ -374,17 +409,14 @@ resources:
...
-                                <attribute name="description">Pink rabbit</attribute>
-                            </attribute>
                         </attribute>
                     </attribute>
                 </attribute>
             </collectionOperation>
         </collectionOperations>
     </resource>
 </resources>
+
```
Comment (alanpoulain) [inline core/openapi.md:419]:
```suggestion
</resources>
```

File: core/openapi.md [hunk 3]
```diff
@@ -311,58 +311,93 @@ You also have full control over both built-in and custom operations documentatio
 
 [codeSelector]
 
+```php
```
Comment (alanpoulain) [inline core/openapi.md:314]:
```suggestion
```php
<?php
// api/src/Entity/Rabbit.php

```

**Comment by alanpoulain** [comment]
Hello, thank you for this PR :slightly_smiling_face:

File: core/openapi.md (hunk 1 above)
Comment (yivi) [reply core/openapi.md:384]: Likely I got it wrong in copy and paste. In my own projects, it's the only thing I have: `<resources xmlns="https://api-platform.com/schema/metadata">`. But trying to match the docs fuller style it seems I messed it up. I'll correct it in a second.

File: core/openapi.md [hunk 4]
```diff
@@ -374,17 +411,14 @@ resources:
...
-                                <attribute name="description">Pink rabbit</attribute>
-                            </attribute>
                         </attribute>
                     </attribute>
                 </attribute>
             </collectionOperation>
         </collectionOperations>
     </resource>
 </resources>
+
```
Comment (alanpoulain) [inline core/openapi.md:421]:
```suggestion
    </resource>
</resources>
```

**Review by alanpoulain — APPROVED** [review]

**Comment by alanpoulain** [comment]
Thank you @yivi!

**Pull request merged**